docs/tutorial/advanced/mkda_association.md

```diff
@@ -0,0 +1,92 @@
+
+Conceptually, this tests if there's evidence of a *population level* association between the task or psychological construct in our meta-analysis and brain activation (for every voxel).
+
+:::info   
+**What happened to the "forward inference" and "reverse inference" analysis?**
+
+On Neurosynth.org, we renamed the pre-generated forward and reverse inference maps; they're now referred to as the "uniformity test" and "association test" maps, performed by the MKDA Chi-Squared algorithm.
+
+Although the method we used hasn't change,d the latter names more accurately capture what these maps actually mean. It was a mistake on our part to have used the forward and reverse inference labels; those labels should properly be reserved for probabilistic maps generated via a Bayesian estimation analysis, rather than for z-scores resulting from frequentist inferential tests. These maps are more difficult to interpret and use correctly, which is why we don't currently support this approach.
+
+See "Further Reading" below to read more about why this change was made
+:::
+
+## Specifying MKDA Chi-Squared Meta-Analysis
+
+Specifying an MKDA Chi-Square meta-analysis in Neurosynth is easy. Simply, select a target set of studies as you would for any other meta-analysis, using either automated or manual selection methods. 
+
+In Step 3 ("Create Meta-Analysis Specification") of your Project, select *MKDAChi2* as the *algorithm*. 
+
+
+![MKDA Chi Squared](/tutorial/mkda_chi_squared_algo.png). 
+
+Next, select the annotation inclusion column you want to use, as before (by default, the "included" column will be used).
+
+Now, select a reference dataset from the dropdown list below. The Neurosynth dataset represents the latest release of the legacy *Neurosynth* dataset (version 7), released July, 2018. The *Neurostore* dataset represents the latest update of our continuously updating "live" dataset, spanning over 20,000 neuroimaging studies. 
+
+![MKDA Chi Squared Reference](/tutorial/mkda_chi_squared_reference.png). 
+
+Now simply complete the rest of the meta-analysis specification wizard to finish. 
+
+## Executing your analysis
+
+The *MKDA Chi-Squared* algorithm is more computational expensive than a traditional *MKDA Density* analysis. As a result, it's unlikely to complete with the freely available resources available on Google Colab.
+
+You can run this workflow locally in one line using docker by copying the command your screen. See the *execution* documentation page for more information. 
+
+:::tip 
+We recommend at least 32GB of RAM to perform a MKDA Chi-Squared analysis
+:::
+
+## Interpreting results
+
+![Neurosynth Maps](/tutorial/neurosynth_paper_fig2.jpg). 
```

Comment:
I believe these are posterior probability maps, so it might be misleading to show this figure since you're focusing on the z-statistic maps in the tutorial.

Reply:
Good point